react-charts/bullet-chart/step2.md

```diff
@@ -1,24 +1,43 @@
-We'll begin by adding a CSS selector to define the chart size...
-
-- Katacoda is setting up a new React application for you. You'll be ready to code once the server starts and you can see "Welcome to PatternFly" on the lower pane.
-
-## Task
-
-1) Navigate to the `src` folder and open `src/app.css`{{open}}
-
-2) Next we'll add some CSS to define the overall chart size
-
-Copy the following code into the app.css file, replacing all of the content there:
-
-<pre class="file" data-filename="src/app.css" data-target="replace">
-.chart-container {
-  height: 150px;
-  width: 600px;
+Create a simple bullet chart.
+
+1. <strong>Navigate to the `src` folder and open `src/App.js`{{open}}</strong>
+
+2. <strong>Set up the general structure of the chart and the necessary import statements.</strong>
+
+Copy the following code into the App.js file, replacing all of the content there:
```

Comment:
This needs tics around `App.js`.

![Screen Shot 2019-12-09 at 10 37 52 AM](https://user-images.githubusercontent.com/17481322/70449494-163d5900-1a70-11ea-8a5b-2fc5e70120aa.png)